### DISCUSSION TITLE
[crud-service] Cache mechanism

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#299] [Q&A] [answered]
### Which section?

Marketplace

### Which is the version?

v11

### Description

Hi guys,
I didn't find anything in the official documentation about that.

Does the `crud-service` have some caching mechanism out of the box?

Thanks.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, no it does not have any kind of cache mechanism

**Reply by frengooo** [reply]
Ok, thanks for your reply @fredmaggiowski.
